src/routes/apiRoute.js

```diff
@@ -0,0 +1,14 @@
+import * as express from 'express';
+import routes from './';
+import controllers from '../controllers';
```

Comment:
The same with controllers. You can import destructured controllers, that way the further code would be shorter and easier to read.

Reply:
For routes, I have made the index file as json but I am not sure how to do for controllers.